packages/ui/src/ImportScreen.tsx

```diff
@@ -165,6 +165,7 @@ export default function ImportScreen({
         { value: "rocketpool", label: "Rocketpool" },
```

Comment:
Shouldnt be contemplated somewhere to dont allow to choose tag stader if network does not support stader?

Reply:
True, stader should only be available for prater and mainnet

Reply:
Okay, it seems like this is already handled:
```
const tagSelectOptions: TagSelectOption[] = ["gnosis", "lukso"].includes(
    network
  )
    ? [{ value: "solo", label: "Solo" }]
    : ["holesky"].includes(network)
    ? [
        { value: "solo", label: "Solo" },
        { value: "rocketpool", label: "Rocketpool" },
        { value: "stakehouse", label: "StakeHouse" },
      ]
    : [
        { value: "solo", label: "Solo" },
        { value: "rocketpool", label: "Rocketpool" },
        
        { value: "stakehouse", label: "StakeHouse" },
        { value: "stakewise", label: "Stakewise" },
        { value: "stader", label: "Stader" },
      ];
```

Reply:
LGTM